crates/vhost/src/vhost_user/message.rs

```diff
@@ -981,38 +990,32 @@ mod tests {
 
     #[test]
     fn check_master_request_code() {
-        let code = MasterReq::NOOP;
-        assert!(!code.is_valid());
-        let code = MasterReq::MAX_CMD;
-        assert!(!code.is_valid());
-        assert!(code > MasterReq::NOOP);
+        assert!(!MasterReq::is_valid(MasterReq::NOOP as _));
+        assert!(!MasterReq::is_valid(MasterReq::MAX_CMD as _));
+        assert!(MasterReq::MAX_CMD > MasterReq::NOOP);
         let code = MasterReq::GET_FEATURES;
-        assert!(code.is_valid());
+        assert!(MasterReq::is_valid(code as _));
         assert_eq!(code, code.clone());
-        let code: MasterReq = unsafe { std::mem::transmute::<u32, MasterReq>(10000u32) };
-        assert!(!code.is_valid());
+        assert!(!MasterReq::is_valid(10000));
     }
 
     #[test]
     fn check_slave_request_code() {
-        let code = SlaveReq::NOOP;
-        assert!(!code.is_valid());
-        let code = SlaveReq::MAX_CMD;
-        assert!(!code.is_valid());
-        assert!(code > SlaveReq::NOOP);
+        assert!(!MasterReq::is_valid(MasterReq::NOOP as _));
+        assert!(!MasterReq::is_valid(MasterReq::MAX_CMD as _));
+        assert!(MasterReq::MAX_CMD > MasterReq::NOOP);
```

Comment:
In this code block, we should use `SlaveReq::` methods/values

Reply:
Fixed.